perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2215,8 +2215,24 @@ public void sendPasswordResetLinkEmail(PerunSession sess, Member member, String
 			throw new InternalErrorException(ex);
 		}
 
+		String subject;
+		try {
+			Attribute a = perunBl.getAttributesManager().getAttribute(sess, language, "urn_perun_entityless_attribute_def_def_nonAuthzPwdResetMailSubject_namespace");
```

Comment:
`a` does not look like a good variable name.

Reply:
Also, you are using attriubte urn wrongly here. Proper user:`"urn:perun:entityless:attribute-def:def:nonAuthzPwdResetMailSubject:" + namespace`